Marlin/src/lcd/menu/menu_configuration.cpp

```diff
@@ -219,6 +219,19 @@ static void lcd_factory_settings() {
 
 #endif
 
+#if ENABLED(TOUCH_MI_PROBE)
+  void menu_touchmi() {
+    START_MENU();
+    ui.defer_status_screen();
+    MENU_BACK(MSG_CONFIGURATION);
+    MENU_ITEM(gcode, MSG_TOUCHMI_INIT, PSTR("M851 Z0\nM500\nG28\nS211 S0\nG1 F200 Z0"));
```

Comment:
Please don't put `M500` here. We don't want things writing to EEPROM without a specific action to write the EEPROM by the user because it writes all current settings, which may cause surprises on the next reboot.